element-hq/element-x-android
# fix linkify

## Type of change


## Content


Ensure message with body and without formatted body got working links.
Also ensure email addresses are linkified.

## Motivation and context


Closes #2038

## Screenshots / GIFs



## Tests



- Send a message with just a raw link and nothing else
- Observe that with this change the link in the message is rendered as a link and is clickable

## Tested devices

- OS version(s):

## Checklist
